kfs-core/src/main/java/edu/arizona/kfs/sys/KFSConstants.java

```diff
@@ -64,4 +64,15 @@
     	public static String PAYMENT_METHOD_EDIT_MODE = "paymentMethodEditMode";
     }
 
+    public static class GeneralErrorCorrectionDocumentConstants {
+    	public static final String GENERATE_ERROR_CERTIFICATION_STMT_ID = "GenerateErrorCertStmt";
+    }
+    
+    public static class ErrorCertificationConstants {
+    	public static final int NUM_ERROR_CERT_FIELDS = 4;

```

Comment:
Hey, I like this! Looks like you just forgot delete the local var referenced in my earlier comment :)